.github/workflows/build-test-package-python.yml

```diff
@@ -47,27 +51,27 @@ jobs:
 
     - name: 'Fetch build script'
       run: |
-        ITK_PYTHON_PACKAGE_GIT_TAG=${{ inputs.itk-python-package-tag }}
-        if [[ -z ${ITK_PYTHON_PACKAGE_GIT_TAG} ]]; then
-          ITK_PYTHON_PACKAGE_GIT_TAG="master"
-        fi
-
-        curl -L https://raw.githubusercontent.com/InsightSoftwareConsortium/ITKPythonPackage/${ITK_PYTHON_PACKAGE_GIT_TAG}/scripts/dockcross-manylinux-download-cache-and-build-module-wheels.sh -O
+        IPP_DOWNLOAD_GIT_TAG=${{ inputs.itk-python-package-tag }}
+        IPP_DOWNLOAD_ORG=${{ inputs.itk-python-package-org }}
+        curl -L https://raw.githubusercontent.com/${IPP_DOWNLOAD_ORG:=InsightSoftwareConsortium}/ITKPythonPackage/${IPP_DOWNLOAD_GIT_TAG:=master}/scripts/dockcross-manylinux-download-cache-and-build-module-wheels.sh -O
         chmod u+x dockcross-manylinux-download-cache-and-build-module-wheels.sh
 
     - name: 'Build 🐍 Python 📦 package'
       shell: bash
       run: |
         export ITK_PACKAGE_VERSION=${{ inputs.itk-wheel-tag }}
+        export ITKPYTHONPACKAGE_TAG=${{ inputs.itk-python-package-tag }}
+        export ITKPYTHONPACKAGE_ORG=${{ inputs.itk-python-package-org }}
         if [ -z ${{ inputs.cmake-options }}]; then
           CMAKE_OPTIONS=""
         else
           CMAKE_OPTIONS="--cmake_options ${{ inputs.cmake-options }}"
         fi
 
-        for tarball in "-manylinux_2_28" "-manylinux2014"; do
+        for manylinux_version in "_2_28" "2014"; do
```

Comment:
Why do we repeat the builds twice?

Reply:
I'm not sure I understand? Here we build once for `-manylinux_2_28` and then once for `-manylinux2014`.

Reply:
Yes, but why do we build for two Linux versions?

Reply:
I believe these are for compatibility with different toolchains, i.e. `-manylinux2014` for older platforms and `-manylinux_2_28` for newer platforms. See the compliance chart in the `manylinux` README: https://github.com/pypa/manylinux#manylinux

@thewtex may be able to provide more detail on the motivation for `-manylinux2014` wheels. I will also update this list in the near future for a third Linux build targeting ARM architectures to build on @thewtex 's recent work in ITKPythonPackage.